Userland/Libraries/LibWeb/PixelUnits.h

```diff
@@ -414,11 +419,23 @@ constexpr Web::CSSPixels round(Web::CSSPixels const& value)
     return ceil(value - Web::CSSPixels::from_raw(Web::CSSPixels::fixed_point_denominator >> 1 /* 0.5 */));
 }
 
+inline Web::CSSPixels sqrt(Web::CSSPixels const& value)
+{
+    return Web::CSSPixels::nearest_value_for(sqrt(value.to_float()));
```

Comment:
prefer `AK::sqrt` or `sqrtf` for floats
also unfortunate that this requires a cast-round-trip
not sure if integral sqrt implementations would be faster

Reply:
I did try my hand at implementing that, but it seemed quite tricky to benchmark properly, I got a lot of inconsistent results. Sometimes running the integral method alongside the cast-round-trip would complete faster than just the integral, optimizer be optimizing I guess.

It did seem like the only methods that would produce faster results than the builtins (which presumably use floating-point instructions) would be approximate, though, so we don't really get any benefit by doing it that way. If I can figure out how to properly benchmark it, though, I can maybe take another stab at ensuring that we use the fastest method.